tests/functional/specs/llm/accept_suggestions.cy.ts

```diff
@@ -0,0 +1,54 @@
+import { LabelStudio, useTextarea } from '@heartexlabs/ls-test/helpers/LSF';
+import {
+  LLM_WHAT_DO_YOU_SEE, LLM_WHAT_ELSE_DO_YOU_SEE,
+  llmTextareaConfig,
+  llmTextareaData,
+  llmTextareaSuggestions
+} from 'data/lmm/accept_suggestions';
+import { FF_DEV_1284, FF_LLM_EPIC } from '../../../../src/utils/feature-flags';
+
+function getLLMSuggestions(area, connectedRegions) {
+  const prompt = area?.results?.[0]?.value?.text?.[0];
+
+  if (prompt) {
+    const suggestions = llmTextareaSuggestions[prompt];
+
+    setTimeout(() => {
+      area.store.loadSuggestions(new Promise((resolve) => setTimeout(() => {
+        resolve(suggestions);
+      }, 1000)), x => x);
+    }, 1000);
```

Comment:
why so long? can we reduce it to make it go faster?

Reply:
We surely can, but I don't know if it catch the problem when it appears.